frontend/ui/widget/opdsbrowser.lua

```diff
@@ -534,7 +546,7 @@ function OPDSBrowser:showDownloads(item)
             if acquisition then
                 local format = self.formats[acquisition.type]
                 if format then
-                    button.text = format
+                    button.text = format .. "\xE2\xAC\x87"
```

Comment:
If you're going to use escaped sequences (it works just fine without but I suppose I shouldn't do that for string comparison reasons or something? :-P) I think they should definitely be commented with info that makes it possible to read the code without having to run it or look things up. I'd probably go for the whole shebang but that might be a bit over the top.

```
-- DOWNWARDS BLACK ARROW ⬇ U+2B07
```